packages/adyen-integration/src/adyenv3/adyenv3-payment-strategy.ts

```diff
@@ -71,10 +72,9 @@ export default class Adyenv3PaymentStrategy implements PaymentStrategy {
 
         const paymentMethod = this._paymentIntegrationService
             .getState()
-            .getPaymentMethodOrThrow(options.methodId);
-        const {
-            initializationData: { environment, clientKey, paymentMethodsResponse },
-        } = paymentMethod;
+            .getPaymentMethodOrThrow<AdyenPaymentMethodInitializationData>(options.methodId);
+        const { environment, clientKey, paymentMethodsResponse } =
+            paymentMethod.initializationData || {};
```

Comment:
Could it return an empty object now?

Reply:
this default `{}` for fixing issue with tsLint, because `initializationData` alwais was as optional property for [PaymentMethod](https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/payment-integration-api/src/payment/payment-method.ts) and can be undefined in some cases